src/components/histogram/progress.tsx

```diff
@@ -0,0 +1,23 @@
+
+type Props = {
+  index: number,
+  size: number,
+  label: string
+}
+
+export const Progress = ( {index, size, label}: Props ) => {
+  return(
+    <div className="flex justify-left m-1 bg-blue-200">
```

Comment:
Style edit - remove background color here:
```suggestion
    <div className="flex justify-left m-1">
```